Marlin/src/module/temperature.cpp

```diff
@@ -1269,56 +1273,58 @@ void Temperature::init() {
 #if ENABLED(FAST_PWM_FAN)
 
   void Temperature::setPwmFrequency(const uint8_t pin, int val) {
```

Comment:
Shouldn't the pin parameter have the `pin_t` type?

Reply:
Probably